src/pages/Pool/PositionPage.tsx

```diff
@@ -326,11 +336,31 @@ const useInverter = ({
   }
 }
 
+const environment = process.env.REACT_APP_VIOLET_ENV
+const clientId = process.env.REACT_APP_VIOLET_CLIENT_ID
```

Comment:
Can this be placed in a shared file so we don't have to redefine it here?

Reply:
I think maybe the focus of change could be in the `baseUrlByEnvironment(environment.toString())` function. Since it seems like it's only used by this function maybe we can make the environment look-up implicit in this function instead?

Reply:
But agree on the clientId one